src/features/click-to-load.js

```diff
@@ -842,9 +855,10 @@ async function replaceClickToLoadElements (targetElement) {
  * the page.
  * @param {unblockClickToLoadContentRequest} message
  * @see {@link ddg-ctp-unblockClickToLoadContent-complete} for the response handler.
+ * @return {Promise<any>}
```

Comment:
Think it's `Promise<None>`?

Reply:
Does TS understand the type return? We could omit entirely if it does.